docs/src/integration.md

```diff
@@ -0,0 +1,63 @@
+# Integration
```

Comment:
This is great, perhaps just rename Integration -> Integration**s**

Reply:
Done